.github/workflows/upload-sbom.yaml

```
@@ -0,0 +1,24 @@
+name: Syft SBOM Creation
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  upload-sbom:
+    runs-on: ubuntu-20.04
+    permissions:
+      contents: read
+      security-events: write
+
+    steps:
+      - uses: actions/checkout@v2
```

We should use `v3`

```suggestion
      - uses: actions/checkout@v3
```

I’ll fix the other workflows!